src/ajax.js

```diff
@@ -861,4 +861,12 @@ jQuery.each( [ "get", "post" ], function( _i, method ) {
 	};
 } );
 
+jQuery.ajaxPrefilter( function( s ) {
+	for ( var i in s.headers ) {
```

Comment:
Our code style requires separate variable declarations at the beginning of the function. So:
```suggestion
	var i;
	for ( i in s.headers ) {
```